aw_qt/manager.py

```diff
@@ -135,6 +138,24 @@ def start(self, testing: bool) -> None:
         exec_cmd = [str(self.path)]
         if testing:
             exec_cmd.append("--testing")
+
+        if self.name.startswith("aw-server"):
+            custom_watcher_static_directories = dict()
+
+            logger.info(f"Searching for watchers with page support...")
+
+            for module in manager.modules:
+                static_dir, name = module.static_directory, module.name
+
+                if name.startswith("aw-watcher"):
+                    if static_dir is not None:
+                        logger.info(f" - Found page support for watcher {name}")
+                        custom_watcher_static_directories[name] = str(module.static_directory.absolute())
+                    else:
+                        logger.info(f" - No static folder found in {name}")
+
+            exec_cmd.append("--custom-watcher-visualizations")
+            exec_cmd.append(json.dumps(custom_watcher_static_directories))
```

Comment:
To @johan-bjareholt's point: This doesn't belong in aw-qt.

aw-server should not have configuration magically applied by aw-qt (it should behave identically when run standalone and when run by aw-qt). Let it only read from the config instead.

Removing this also removes the added dependency `Module -> Manager`, which I think is important to avoid an unnecessary bidirectional class dependency.

Since this "autoconfig" is the only real feature in this PR, it makes the entire PR unneeded (a good thing!).

Reply:
I thought the "autoconfig" was convenient, but I totally understand your point. You can close the aw-qt pull request if you want.